kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/FeatureService.java

```diff
@@ -20,9 +20,11 @@
 @Log4j2
 public class FeatureService {
 
-  private final BrokerService brokerService;
+  private static final String DELETE_TOPIC_ENABLED_SERVER_PROPERTY = "delete.topic.enable";
+
+  private final AdminClientService adminClientService;
 
-  public Mono<List<Feature>> getAvailableFeatures(KafkaCluster cluster) {
+  public Mono<List<Feature>> getAvailableFeatures(KafkaCluster cluster, @Nullable Node controller) {
```

Comment:
Not. sure that adding controller argument here is the right place. Maybe it should be wrapped inside adminClientService?

Reply:
@germanosin idea was to minimize api calls count. In the caller code we already have cluster description info, so, no need to query it again. Maybe we should pass whole ClusterDescription instead of only controller node?